### ISSUE TITLE
Upgrade description for greyed out and upgraded icons

### ISSUE THREAD

**Issue by aeoncleanse** [FAForever/fa#1264]
It annoys me that, when an upgrade is greyed out or you already have the slot maxed, you can't see the tooltip describing it. I'd like this enabled please.

**Comment by Uveso** [comment]
Hello icedreamer.
Do you mean the tooltips on a ACU or on a unit ? 
Well here are both solutions :)

Open construction.lua,
go to line 777 and change the code from:
```LUA
            if control.Data.Disabled then
                control:Disable()
                if not control.Data.Selected then
                    control.Icon:SetSolidColor('aa000000')
                end
            else
                control:Enable()
            end
```
and change it to:
```LUA
            if control.Data.Disabled then
                --control:Disable()
				control.Data.TooltipOnly = true
                if not control.Data.Selected then
                    control.Icon:SetSolidColor('aa000000')
                end
            else
				control.Data.TooltipOnly = false
                control:Enable()
            end
```
Now go to line 1285 and change it from:
```LUA
elseif item.type == 'enhancement' then
```
to:
```LUA
    elseif item.type == 'enhancement' and button.Data.TooltipOnly == false then
```
Now you can see enhancement tooltips on disabled buttons :)
Testet with normal ACU's and Black Ops advanced ACU.

if u want to see the tooltip, on normal units, like shields etc,
Just go to line 1057 and change it from:

```LUA
if item.type == 'item' then

```
to:
```LUA
if item.type == 'item' or item.type == 'unitstack' then

```

Testet it only 10 minutes, but seems to work.
Can i do anything else for you ? :D

**Comment by aeoncleanse** [comment]
You know, isn't it about time you started submitting PRs for this stuff? ;)

**Comment by Uveso** [comment]
I know, you are right :)
I was thinking about that, while writing the line numbers. PR might be easier for both sides :)
But you know i am slow!
Will starting with the github manual first :dancer:

**Comment by duk3luk3** [comment]
hey @Uveso, I turned your changes into [a PR](https://github.com/FAForever/fa/pull/1269). Here's what I did:

1. `git checkout develop` to get to develop branch
2. `git pull upstream develop` to update the branch - upstream is the name of the `git remote` that points at the main repo
3. `git checkout -b fix/disabledbuttontooltips` - create a new branch off of develop
4. Edit the files
5. `git commit -a` to commit the changes
6. `git push origin fix/disabledbuttontooltips` to push the branch to my fork repo
7. Go to `https://github.com/FAForever/fa` and click on "Compare & Pull Request" under "You recently pushed branches"
8. Click "Commit Pull Request"

And done!
